### ISSUE TITLE
ioError [DioErrorType.DEFAULT]: FormatException: Invalid character in cookie value, code unit: '32' (at character 5)

### ISSUE THREAD

**Issue by Shwise** [flutterchina/dio#785]
### New Issue Checklist

- [x] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |flutter      |
| Platform Version               | e.g.  1.5.0 / 12.0 / 9.0                              |latest      |
| Dio Version                    | e.g.  2.1.0 / 1.0.17                                  | latest     |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |N/A      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   | 100%     |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |N/A      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Dio (with cookie jar plugin) get request reports "Invalid character in cookie value" but references a header that is not a cookie at all, but "last-modified".

**Comment by woelmer** [comment]
This is still an issue for me

**Comment by effusiveperiscope** [comment]
This is also an issue for me.
